lib/launcher.rb

```diff
@@ -10,6 +10,15 @@ class Launcher
 
     attr_reader :settings
 
+    RACK_SERVERS = {
+      :webrick => Rack::Handler::WEBrick,
+      :puma => Rack::Handler::Puma
```

Comment:
won't this will require both gems to load? e.g. if you dont have puma gem, this line will break?

Reply:
Does it hurt to load them both? If it don't let's ignore it? If it does, then let's definitely take care of this.

Reply:
So it looks like webrick is always present, dependency of sinatra or something. But we should be able to install foreman-proxy without puma bundler group, so this needs to be taken care of.

Reply:
so instead of the original conversation, you want it more plugable?
that is, if it present, then load all of it requirements?